packages/mermaid/src/diagrams/info/info.spec.ts

```diff
@@ -1,24 +1,17 @@
-// @ts-ignore - jison doesn't export types
-import { parser } from './parser/info.jison';
-import { db } from './infoDb.js';
-
-describe('info diagram', () => {
-  beforeEach(() => {
-    parser.yy = db;
-    parser.yy.clear();
-  });
+import { parser } from './infoParser.js';
 
+describe('info', () => {
   it('should handle an info definition', () => {
     const str = `info`;
-    parser.parse(str);
-
-    expect(db.getInfo()).toBeFalsy();
+    expect(() => {
+      parser.parse(str);
+    }).not.toThrow();
   });
 
   it('should handle an info definition with showInfo', () => {
```

Comment:
Can you add a failure case also, to verify the error messages.

Reply:
How deep should we verify? I wouldn't recommend to match exact error message because when updating `chevrotain` or `langium` might have some differences.

Reply:
Let's match it for now and modify it if has been changed.